pkgs/desktops/pantheon/desktop/elementary-greeter/default.nix

```diff
@@ -89,12 +89,6 @@ stdenv.mkDerivation rec {
   ];
 
   patches = [
-    # Upstream code not respecting our localedir
-    # https://github.com/elementary/greeter/pull/545
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
https://github.com/NixOS/nixpkgs/pull/139163